libraries/CryptoInterface/examples/HelloCrypto/HelloCrypto.ino

```diff
@@ -0,0 +1,102 @@
+/**
+   This example shows the functionality of the CryptoInterface library.
+*/
+
+#include <ESP8266WiFi.h>
+#include <TypeConversion.h>
+#include <CryptoInterface.h>
+
+namespace TypeCast = esp8266::TypeConversion;
+using namespace experimental;
+
+/**
+   NOTE: Although we could define the strings below as normal String variables,
+   here we are using PROGMEM combined with the FPSTR() macro (and also just the F() macro further down in the file).
+   The reason is that this approach will place the strings in flash memory which will help save RAM during program execution.
+   Reading strings from flash will be slower than reading them from RAM,
+   but this will be a negligible difference when printing them to Serial.
+
+   More on F(), FPSTR() and PROGMEM:
+   https://github.com/esp8266/Arduino/issues/1143
+   https://arduino-esp8266.readthedocs.io/en/latest/PROGMEM.html
+*/
+constexpr char masterKey[] PROGMEM = "w86vn@rpfA O+S"; // Use 8 random characters or more
+
+void setup() {
+  // Prevents the flash memory from being worn out, see: https://github.com/esp8266/Arduino/issues/1054 .
+  // This will however delay node WiFi start-up by about 700 ms. The delay is 900 ms if we otherwise would have stored the WiFi network we want to connect to.
+  WiFi.persistent(false);
+
+  Serial.begin(115200);
+  delay(50); // Wait for Serial.
```

Comment:
Not needed on this chip.  I think it's a leftover from the original AVR.

Reply:
To be specific, the `delay` isn't needed.  `Serial.begin` still is.

Reply:
I've previously noticed that initial prints are sometimes dropped without the delay. This has not always been the case though, so maybe it was a bug that has been fixed now?

Reply:
It could have been fixed before I started working on the core, I don't really know.  But for several years now it's been fine to send/receive immediately after a `Serial.begin()`.  All other examples that I know of have removed the delay...

Reply:
Hmm... I do remember having problems with the first few `Serial.println` not showing up as late as the autumn of 2019, but removing the `delay` now seems to cause no issue. Perhaps it only happens in some special cases, or perhaps the problem has been fixed. I'll remove the `delay` for now since it does not seem to have an effect.